src/main/java/com/coherent/training/selenium/stanila/pages/DynDataLoadingPOM.java

```diff
@@ -0,0 +1,28 @@
+package com.coherent.training.selenium.stanila.pages;
+
+import com.coherent.training.selenium.stanila.base.BaseClass;
+import lombok.SneakyThrows;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class DynDataLoadingPOM extends BaseClass {
+    @FindBy(xpath = "//button[@id='save']")
+    private WebElement newUser;
+    @FindBy(xpath = "//div[@id='loading']")
+    private WebElement response;
+    public DynDataLoadingPOM(){
+        PageFactory.initElements(driver, this);
+    }
+
+    @SneakyThrows
+    public void getNewUserMeth(){
```

Comment:
get means that you want to return something

Reply:
Modified in clickOnNewUser()